### ISSUE TITLE
confusing start if user is not using a transpiler

### ISSUE THREAD

**Issue by bbrooks** [jenius/stockfighter-node#2]
For those of us who are new to node and aren't using a transpiler, I think it might be useful to add this alternate import method to the docs:

``` js
stockfighter = require('stockfighter');
API = stockfighter.default;
```

I am new to node and came upon this method by hacking, so feel free to recommend a more elegant solution.

**Comment by jescalan** [comment]
Hey, I'm so sorry about that, I was not even thinking about it. I'll make sure that it exports normally and push a new version in a moment :grinning:

**Comment by jescalan** [comment]
Resolved in `stockfighter@0.0.2` :+1:
